src/sentry/static/sentry/app/plugins/components/issueActions.jsx

```diff
@@ -68,6 +80,86 @@ class IssueActions extends PluginComponentBase {
     );
   }
 
+  setDependentFieldState(fieldName, state) {
+    const dependentFieldState = {...this.state.dependentFieldState};
+    dependentFieldState[fieldName] = state;
+    this.setState({dependentFieldState});
+  }
+
+  loadOptionsForDependentField = async field => {
+    const formData = this.getFormData();
+
+    const url =
+      '/issues/' +
+      this.getGroup().id +
+      '/plugins/' +
+      this.props.plugin.slug +
+      '/options/';
+
+    //find the fields that this field is dependent on
+    const dependentFields =
+      fromPairs(field.depends.map(fieldKey => [fieldKey, formData[fieldKey]])) || {};
```

Comment:
Instead of using lodash you can use [Object.fromEntries()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries) (it's polyfilled)